# SPDX-FileCopyrightText: Huawei Inc.
#
# SPDX-License-Identifier: Apache-2.0

Patch for //developtools/hdc_standard git repository of OpenHarmony 3.1 codebase.

Disable hdc_std since there is a separate recipe for the host tool.

Signed-off-by: Kristijonas Ciulcinskas <kristijonas.ciulcinskas@huawei.com>
Signed-off-by: Grzegorz Gwóźdź <grzegorz.gwozdz@huawei.com>
Upstream-Status: Inappropriate [configuration/integration]

diff --git a/developtools/hdc_standard/BUILD.gn b/developtools/hdc_standard/BUILD.gn
index b2c1cfbeea4a..8b4eeaa1fc1f 100644
--- a/developtools/hdc_standard/BUILD.gn
+++ b/developtools/hdc_standard/BUILD.gn
@@ -188,32 +188,12 @@ ohos_executable("hdc_std") {
 
 group("hdc_target_standard") {
   deps = [
-    ":hdc_std",
     ":hdcd",
   ]
 }
 
-group("hdc_target_standard_linux") {
-  deps = [ ":hdc_std(//build/toolchain/linux:clang_x64)" ]
-}
-
-group("hdc_target_standard_mingw") {
-  deps = [ ":hdc_std(//build/toolchain/mingw:mingw_x86_64)" ]
-}
-
 group("hdc_target_standard_all") {
-  deps = [
-    ":hdc_target_standard",
-    ":hdc_target_standard_linux",
-    ":hdc_target_standard_mingw",
-  ]
-}
-
-group("hdc_target_standard_windows") {
-  deps = [
-    ":hdc_target_standard",
-    ":hdc_target_standard_mingw",
-  ]
+  deps = [ ":hdc_target_standard" ]
 }
 
 group("hdc_all") {